src/merkle/smt/full/tests.rs

```diff
@@ -258,6 +258,195 @@ fn test_smt_removal() {
     }
 }
 
+/// This tests that we can correctly calculate prospective leaves -- that is, we can construct
```

Comment:
typo: tests -> test's

Reply:
The test isn't possessing anything, so there's no need for an apostrophe here.